### ISSUE TITLE
Getting attribution scores for unlabelled data.

### ISSUE THREAD

**Issue by gauneg** [YRdddream/attattr#10]
Hi,
In the code for attribution score calculation in the file `generate_attrscore.py`, if we wanted to get the score using say `SST-2` fine-tuned model. If we generate the attribution score for an input text without gold labels, i.e. making the assumption that $\text{predicted labels} = \text{true label}$. What would be the implications of it, would this affect the attribute tree generation so much as to render this approach unusable?

Referring to the file `generate_attrscore.py`:

```python
tar_prob, grad = model(input_ids, segment_ids, input_mask, label_ids, tar_layer, one_batch_att, pred_label=pred_label)
```
for $\text{predicted labels} = \text{true label}$, changing it to:

```python
pred_tensor = torch.argmax(baseline_logits)
tar_prob, grad = model(input_ids, segment_ids, input_mask, pred_tensor.unsqueeze(0), tar_layer, one_batch_att, pred_label=pred_label)
```

**Comment by YRdddream** [comment]
Hi, 
The variable "label_ids" is an argument passed to the forward function but it is not used for computing attribution scores. We just use the "pred_label" to compute gradients (Line 1033 in model_attr.py), so the results will only be affected by "pred_label" instead of "label_ids". For an unlabeled example, you can replace "label_ids" with "pred_label" or set it to a random label. It will not affect the final results.
